src/cluster.c

```diff
@@ -7475,3 +7750,59 @@ unsigned int countChannelsInSlot(unsigned int hashslot) {
     raxStop(&iter);
     return j;
 }
+
+/* Replicate all migrating and importing slot states to a
+ * a given replica client. */
+void replicateOpenSlots(client *c)
+{
+    if (!server.cluster_enabled) return;
+
+    list *replicas = listCreate();
+    listAddNodeHead(replicas, c);
```

Comment:
I'm pretty sure this isn't doing what you expect, but I think it works OK. replicationFeedSlaves adds the request to the shared buffer, which all replicas get. So we are broadcasting this to all nodes. These are all idempotent so I think it's OK.

Reply:
Agreed that this is not needed. Removing